Framework/Algorithms/src/PolarizationCorrectionWildes.cpp

```diff
@@ -528,6 +540,28 @@ API::WorkspaceGroup_sptr PolarizationCorrectionWildes::groupOutput(const Workspa
   return outWS;
 }
 
+/**
+ * Add an output name in the correct position in the vector and to the ADS.
+ * @param names A list of the names of the workspaces the algorithm has generated.
+ * @param spinStateOrder The order the output should be in.
+ * @param baseName The base name for the output workspaces ("BASENAME_SPINSTATE" e.g "OUTNAME_+-")
+ * @param ws The workspace to add to the vector and ADS.
+ * @param spinState The spin state the workspace represents.
+ */
+void PolarizationCorrectionWildes::addSpinStateOutput(std::vector<std::string> &names,
+                                                      const std::string &spinStateOrder, const std::string &baseName,
+                                                      const API::MatrixWorkspace_sptr &ws,
+                                                      const std::string &spinState) {
+  if (spinStateOrder.empty()) {
+    names.emplace_back(baseName + "_" + spinState);
+    API::AnalysisDataService::Instance().addOrReplace(names.back(), ws);
+  } else {
+    auto const &i = PolarizationCorrectionsHelpers::indexOfWorkspaceForSpinState(spinStateOrder, spinState);
```

Comment:
I encountered a build issue on line 559 that seems to stem from a type mismatch in the call to indexOfWorkspaceForSpinState. The function is expecting a const std::vector<std::string>& as its first argument, but it appears that a std::string is being passed instead